src/metabase/api/session.clj

```diff
@@ -265,7 +265,7 @@
          (if (and user (:is_active user))
            (mw.session/set-session-cookie request response session)
            (throw (ex-info (str disabled-account-message)
-                           {:status-code 400
+                           {:status-code 401
```

Comment:
This was already returning `401` because of the `http-401-on-error` wrapper, but I'm making this change for consistency.